docs/INITALIZE_SDK_AND_CREATE_ACCOUNT.md

```diff
@@ -0,0 +1,34 @@
+### Initialize the SDK and Creating a User’s Kin Account ###
+
+If your app presents Kin Spend and Earn offers to your users, then each user needs a Kin wallet and account in order to take advantage of those offers. During initialization and before any other Kin sdk API calls, your app must call the SDK’s ```Kin.start(…)``` function while passing a unique ID for the current user. If that user already has a Kin account, the function only accesses the existing account. Otherwise, the function creates a new wallet and account for the user.
```

Comment:
One tick, also instead of the three points char (`…`), you probably want to use actual three points (`...`)